frontend/src/utils/helpers/getLayoutBoxes/getLayoutBoxes.ts

```diff
@@ -26,15 +26,22 @@ const getLayoutBoxes = ({
   if (!wrapRef.current) {
     return {};
   }
-  const fixBigRatio = layoutProps.pinnedSubscriberCount > 1;
+  // For multiple pinned participants with no screenshare we make all large tiles landscape
+  const fixLargeTilesLandscape =
```

Comment:
Sounds like "desktopView" or "laptopView" to me :-)

Reply:
hmm, that's not what it is... so this is probably a big name, tbh this is why I avoided trying to make an abstraction because it's not super clear either way.
Another option I guess is to just pass the context into the layout manager rather than the other way around...